### ISSUE TITLE
How to command PolyCoder to generate code?

### ISSUE THREAD

**Issue by Prestapro** [VHellendoorn/Code-LMs#45]
`alex@billing desktop % python3 poly.py           
The cache for model files in Transformers v4.22.0 has been updated. Migrating your old cache. This is a one-time only operation. You can interrupt this and resume the migration later on by calling `transformers.utils.move_cache()`.
0it [00:00, ?it/s]
Downloading (…)lve/main/config.json: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 609/609 [00:00<00:00, 172kB/s]
Downloading (…)model.bin.index.json: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████| 42.0k/42.0k [00:00<00:00, 725kB/s]
Downloading (…)l-00001-of-00002.bin: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████| 9.98G/9.98G [24:28<00:00, 6.79MB/s]
Downloading (…)l-00002-of-00002.bin: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1.26G/1.26G [04:21<00:00, 4.81MB/s]
Downloading shards: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [28:54<00:00, 867.03s/it]
Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:10<00:00,  5.36s/it]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:0 for open-end generation.
def binarySearch(arr, left, right, x):
    mid = (left + right) // 2
    if arr[mid] == x:
        return mid
  
def binarySearch(arr, left, right, x):
    mid = (left + right) // 2
    if arr[mid] > x:
        return binarySearch(arr
def binarySearch(arr, left, right, x):
    mid = (left + right) // 2
    if arr[mid] < x:
        return -1
 
def binarySearch(arr, left, right, x):
    mid = (left + right) / 2
    if arr[mid] > x:
        return binarySearch(arr
alex@billing desktop % python3 poly.py         
Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:17<00:00,  8.63s/it]
alex@billing desktop % 
`

**Comment by urialon** [comment]
Hi @Prestapro ,
Thank you for your interest in PolyCoder!

I can't debug the  outputs that you've sent because you did not provide the `poly.py` file.

In any case, please see https://github.com/VHellendoorn/Code-LMs#october-2022---polycoder-is-available-on-huggingface

Best,
Uri
